server/lib/uploaded-files.ts

```diff
@@ -6,7 +6,7 @@ import { FileKind } from '../models/UploadedFile';
  * replace this by real DB associations.
  */
 export const FileFieldsDefinition: Record<
-  FileKind,
+  Exclude<FileKind, 'AGREEMENT_ATTACHMENT'>,
```

Comment:
I think this attachment type did not fit cleanly with the deletion script because attachment has a direct foreign key to UploadedFile instead of just an url field.

Reply:
Yes, we need to add that logic into the deletion script but it doesn't have to be in this PR